README.md

```diff
@@ -93,6 +93,56 @@ SELECT COUNT(jid) FROM sidekiq_jobs WHERE event = 'error' AND time > now() - 1d
 
 Et cetera.
 
+### Stats and Queues metrics
+
+To collect metrics for task stats and queues, you need to run the script periodically. For example you can use gem `clockwork`.
```

Comment:
It would be nice to have working example code for the user's Clockwork configuration that they could just copy and paste.